src/en/stream.rs

```diff
@@ -45,10 +45,7 @@ impl<'en> Stream for JSONMapEntryStream<'en> {
                 None => Some(Ok(Bytes::from_static(COLON))),
             }
         } else if !this.value.is_terminated() {
-            match ready!(this.value.as_mut().poll_next(cxt)) {
-                Some(result) => Some(result),
-                None => None,
-            }
```

Comment:
this match doesn't actually do anything, since the output of `ready!` is already either `None` or `Some(result)`